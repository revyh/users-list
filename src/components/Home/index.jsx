import React from 'react';

const styles = {
  position: 'absolute',
  top: '50%',
  left: '50%',
  transform: 'translate(-50%, -50%)',
  fontSize: '2em',
};

export default function Home() {
  return <div style={styles}>Welcome to users list app</div>;
}
