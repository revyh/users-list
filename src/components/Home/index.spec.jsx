import React from 'react';
import {shallow} from 'enzyme';
import Home from '.';

it('should render', () => {
  expect(shallow(<Home />)).toMatchSnapshot();
});
