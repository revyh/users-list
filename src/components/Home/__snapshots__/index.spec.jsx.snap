// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  style={
    Object {
      "fontSize": "2em",
      "left": "50%",
      "position": "absolute",
      "top": "50%",
      "transform": "translate(-50%, -50%)",
    }
  }
>
  Welcome to users list app
</div>
`;
